src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
ok. Let me try using injectINtl outside of this component then and passing it in. Thats what you are suggesting?